src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/BulkScanValidationPatterns.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model;
+
+public class BulkScanValidationPatterns {
```

Comment:
it looks like a very good candidate to go to the lib

Reply:
Moved into the BSP Commons Lib as requested